src/mount_efs/__init__.py

```diff
@@ -256,7 +256,31 @@
 WATCHDOG_SERVICE = "amazon-efs-mount-watchdog"
 
 # MacOS instances use plist files. This files needs to be loaded on launchctl (init system of MacOS)
-
+<<<<<<< HEAD
```

Comment:
Same here, not a clean commit

Reply:
Would it help if I reforked and sent another pull request?

Reply:
Yes, that will also help.

Reply:
Also if you don't mind, I can submit the PR instead, which should be pretty quick.